code/datums/datum.dm

```diff
@@ -20,6 +20,7 @@
 // This should be overridden to remove all references pointing to the object being destroyed.
 // Return the appropriate QDEL_HINT; in most cases this is QDEL_HINT_QUEUE.
 /datum/proc/Destroy(force = FALSE, ...)
+	SHOULD_CALL_PARENT(TRUE)
```

Comment:
This here is the magic. During a travis run, dreamchecker will look for these annotations and trace entire call stacks down, failing if you do not call the parent on a proc

Note, this applies all the way down a call stack